### PR TITLE
Minimize the example

### PR DESCRIPTION
**Pull request by bradbenjamin-wf**
I don't want users of this repo to have to rip out so much of the app.

I tend to show this repo to students doing prototype competition to show them how easy server-side can be.  Often when they're building an app in iOS or Android, their focus is best spent on the client-side.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 *.pyc
 .idea/
+*/.svn/
```
Comment (sagersmith8) [inline .gitignore]: I think you probably just want `*.svn`

File: README.md [hunk 2]
```diff
@@ -1,18 +1,48 @@
...
+1. [GAE](https://developers.google.com/appengine/)
+2. [bottle.py](http://bottlepy.org/) for handling requests, these are the @route annotations for assigning URL's to handler methods
+3. [jinja2](http://jinja.pocoo.org/docs/dev/) for HTML templating in static/template/ folder
+
+If you want to go further and add payments, cron jobs, or task queues, you may want to check out the older `full_example` branch.
+
+Setup for Windows
+============
+1. Install Python 2.7: https://www.python.org/downloads/ 
+2. Install App Engine SDK for Python https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngine-1.9.40.msi
```
Comment (sagersmith8) [inline README.md]: 2. Install [App Engine SDK for Python](https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngine-1.9.40.msi)

File: README.md [hunk 3]
```diff
@@ -1,18 +1,48 @@
...
+
+1. [GAE](https://developers.google.com/appengine/)
+2. [bottle.py](http://bottlepy.org/) for handling requests, these are the @route annotations for assigning URL's to handler methods
+3. [jinja2](http://jinja.pocoo.org/docs/dev/) for HTML templating in static/template/ folder
+
+If you want to go further and add payments, cron jobs, or task queues, you may want to check out the older `full_example` branch.
+
+Setup for Windows
+============
+1. Install Python 2.7: https://www.python.org/downloads/
```
Comment (sagersmith8) [inline README.md]: 1. Install [Python 2.7](https://www.python.org/downloads/)

File: README.md [hunk 4]
```diff
@@ -1,18 +1,48 @@
...
+2. [bottle.py](http://bottlepy.org/) for handling requests, these are the @route annotations for assigning URL's to handler methods
+3. [jinja2](http://jinja.pocoo.org/docs/dev/) for HTML templating in static/template/ folder
+
+If you want to go further and add payments, cron jobs, or task queues, you may want to check out the older `full_example` branch.
+
+Setup for Windows
+============
+1. Install Python 2.7: https://www.python.org/downloads/ 
+2. Install App Engine SDK for Python https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngine-1.9.40.msi
+3. Open GoogleAppEngineLauncher and add gae_starter as a project ('gae_starter' app name, /path/to/workspace/ as folder)
```
Comment (sagersmith8) [inline README.md]: Open GoogleAppEngineLauncher and add `gae_starter` as a project (`'gae_starter'` app name, `/path/to/workspace/` as folder)

File: README.md [hunk 5]
```diff
@@ -1,18 +1,48 @@
...
+
+Setup for Mac
 ============
-virtual environment 2.7
-app engine python library
-pip install -r requirements.dev
+1. Install Python 2.7 https://www.python.org/downloads/release/python-2713rc1/
```
Comment (sagersmith8) [inline README.md]: 1. Install [Python 2.7](https://www.python.org/downloads/release/python-2713rc1/) or `brew install python`

File: README.md [hunk 6]
```diff
@@ -1,18 +1,48 @@
...
+2. Install App Engine SDK for Python https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngineLauncher-1.9.49.dmg
```
Comment (sagersmith8) [inline README.md]: 2. Install [App Engine SDK for Python](https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngineLauncher-1.9.49.dmg)

File: README.md [hunk 7]
```diff
@@ -1,18 +1,48 @@
...
+3. Open GoogleAppEngineLauncher and add gae_starter as a project ('gae_starter' app name, /path/to/workspace/ as folder)
```
Comment (sagersmith8) [inline README.md]: 3. Open GoogleAppEngineLauncher and add `gae_starter `as a project (`'gae_starter'` app name, `/path/to/workspace/` as folder)

File: README.md [hunk 8]
```diff
@@ -1,18 +1,48 @@
...
+
+Deploying
+============
+1. You'll need to create a project at https://console.developers.google.com (ex: project_id: abcde-12345)
+2. Change app.yaml application to abcde-12345
+3. Run `python appcfg.py update app.yaml` (you'll have to set path to python and appcfg.py if they aren't sym-linked already)
```
Comment (sagersmith8) [inline README.md]: 3. Run `python appcfg.py update app.yaml` (you'll have to set path to python and `appcfg.py` if they aren't sym-linked already)

File: README.md [hunk 9]
```diff
@@ -1,18 +1,48 @@
...
+4. Visit https://console.cloud.google.com/appengine?project=abcde-12345 to make sure the version you uploaded is the default version.
```
Comment (sagersmith8) [inline README.md]: 4. Visit [https://console.cloud.google.com/appengine?project=abcde-12345](https://console.cloud.google.com/appengine?project=abcde-12345) to make sure the version you uploaded is the default version.

File: lib/README.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+The lib folder is a little confusing.  It is for dependencies, but it has to fill 2 purposes:
+1. Local development dependencies
+2. Supply Dependencies so they are uploaded on deploy, you do not run 'pip install' on the app engine machines
```
Comment (sagersmith8) [inline lib/README.md]: `pip install`

File: lib/README.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
...
+
+When you pip install with a "-t lib/" at the end, it's not going to install to your python site-packages folder, but rather just dump the dependencies to lib/.
```
Comment (sagersmith8) [inline lib/README.md]: `"-t lib/"`

File: src/core.py [hunk 12]
```diff
@@ -0,0 +1,14 @@
+import os
+import jinja2
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__) + "/../static/template"),
```
Comment (sagersmith8) [inline src/core.py]: Maybe add a comment here. I had a really difficult time figuring out how this worked the first time I tried to use this

File: src/main.py [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+
+import bottle
+import json
+from bottle import route, post, template, error, request, response
+from models import get_silly_data
+from core import respond
+from google.appengine.ext.webapp.util import run_wsgi_app
+
+
+
```
Comment (sagersmith8) [inline src/main.py]: Kill this line

File: src/main.py [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+
```
Comment (sagersmith8) [inline src/main.py]: Kill this line too

File: static/css/site.css [hunk 15]
```diff
@@ -0,0 +1 @@
+body {font-family: Lato;}
```
Comment (sagersmith8) [inline static/css/site.css]: Github wants a newline

File: static/template/base.html [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <!-- Le styles -->
+    <link href="/css/minireset.min.css" rel="stylesheet">
+    <link href="/css/site.css" rel="stylesheet">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="shortcut icon" href="/img/favicon.ico">
+    {%block extrastyle %}{%endblock%}
```
Comment (sagersmith8) [inline static/template/base.html]: Maybe a comment on how these things work

**Review by sagersmith8 — COMMENTED** [review]
I wouldn't be super upset if you added javadocs sort of explaining things a little better. Also general comments on how jinja works would probably be useful

File: src/core.py (hunk 12 above)
Comment (bradbenjamin-wf) [inline src/core.py]: You wouldn't have had to figure it out if you'd followed the project format this project started with.  I need to explain templates in general...I think in the readme.

File: static/template/base.html (hunk 16 above)
Comment (bradbenjamin-wf) [inline static/template/base.html]: I need to explain templates in general...I think in the readme.

File: README.md (hunk 2 above)
Comment (bradbenjamin-wf) [inline README.md]: I know how to do links, but they're not all that useful when you want to copy paste directions into Slack or something...if the URL is visible here then it pastes visibly.

**Comment by bradbenjamin-wf** [comment]
@sagersmith8 Comments addressed.

File: README.md (hunk 2 above)
Comment (sagersmith8) [inline README.md]: What about something like this?

2. Install App Engine SDK for Python [https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngine-1.9.40.msi](https://storage.googleapis.com/appengine-sdks/featured/GoogleAppEngine-1.9.40.msi)

**Pull request merged**